src/lib/layout/createProject.svelte

```diff
@@ -94,6 +87,27 @@
                     <Typography.Text>Region cannot be changed after creation</Typography.Text>
                 </Layout.Stack>
             {/if}
+            {#if projectsLimited}
+                {#if isProPlan}
+                    <Alert.Inline status="info" title="Expand for $15/project per month">
```

Comment:
check with Damodar is this amount value is accessible from some backend call or model.